components/AMODashCount.js

```diff
@@ -5,49 +5,43 @@ import DashCount from './DashCount';
 
 export default function AMODashCount(props) {
   const repo = props.repo.replace(/_/g, '-');
-  let warning = false;
+  let warningLimit = null;
```

Comment:
This doesn't need to default to `null` anymore as it's checked against being undefined.